### DISCUSSION TITLE
Why are the smileys so small and some are not displayed?

### DISCUSSION THREAD

**Discussion by master255** [community/community#39741] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Why are the smileys so small and some are not displayed?

**Comment by radar-blip** [comment]
On your profile? I dont see them displayed anywhere. 😄

**Comment by master255** [comment]
Smiles 2010 year vs 2022
![image](https://user-images.githubusercontent.com/5380115/204399760-25def136-bf99-4851-a333-3fecd0cec2a6.png)
![image](https://user-images.githubusercontent.com/5380115/204399853-60b7b0da-5e5b-44cc-aff6-79a532096a1b.png)

**Comment by master255** [comment]
Githab, why are you living in 2010?

**Comment by master255** [comment]
Even stealing someone else's smiley faces would be more dignified than living with such shame.
